### PR TITLE
[semver:major] 2.0

### PR DESCRIPTION
**Pull request by gmemstr**
Major overhaul. Depends on https://github.com/CircleCI-Public/orb-tools-orb/pull/90

### CODE REVIEW

File: src/@orb.yml [hunk 1]
```diff
@@ -1,148 +1,9 @@
...
-            echo "SHELLCHECK_OPTS=$SHELLCHECK_OPTS"
-            <</parameters.ignore>>
-            find '<<parameters.path>>' -not -path '<<parameters.exclude>>' \
-              -type f -name '<<parameters.pattern>>' | \
-              xargs shellcheck --external-sources | \
-              tee -a '<<parameters.output>>'
-      - store_artifacts:
-          path: '<< parameters.output >>'
-          destination: shellcheck
+  homepage: https://shellcheck.net
```
Comment (lokst) [inline src/@orb.yml:9]: I believe this should be `home_url`, to be properly rendered in the orb registry

**Comment by KyleTryon** [comment]
Looks great 👍  @gmemstr to avoid an inception problem, I think we may need to manually publish this version when ready (manually pack/publish) and then update the config later with a semver:skip. That way we can use this version of the orb, in the orb-tools release.

**Comment by KyleTryon** [comment]
Updating functionality/logic

**Comment by KyleTryon** [comment]
I believe we are finally ready 🎉 . Giving final inspections.

**Comment by sbe-arg** [comment]
Good work on everything except changing ignore for exclude XD

**Comment by KyleTryon** [comment]
> Good work on everything except changing var `ignore` for `exclude` XD
> Breaks config from v1.x to v2.x if using `@volatile`
> 
> https://github.com/CircleCI-Public/shellcheck-orb/pull/22/files#diff-ef158f82f8d59ced9c306196050bd9d4R5

Hello,

This is expected. It is highly recommended _not_ to use volatile. Orbs use semver versioning, where a major version change indicates a breaking change.

**Pull request merged**